articles/communication-services/concepts/sms/messaging-policy.md

```diff
@@ -104,16 +104,19 @@ Message recipients may revoke consent and opt-out of receiving future messages t
 
 Ensure that message recipients can opt-out of future messages at any time. You must also offer multiple opt-out options. After a message recipient opts-out, you should not send additional messages unless the individual provides renewed consent.
 
-One of the most common opt-out mechanisms is to include a “STOP” keyword in the initial message of every new conversation. Be prepared to remove customers that reply with a lowercase “stop” or other common keywords, such as “unsubscribe” or “cancel.” After an individual revokes consent, you should remove them from all recurring messaging campaigns unless they expressly elect to continue receiving messages from a particular program.
+For example one of the most common opt-out mechanisms in SMS applications is to include a “STOP” keyword in the initial message of every new conversation. Be prepared to remove customers that reply with a lowercase “stop” or other common keywords, such as “unsubscribe” or “cancel.” After an individual revokes consent, you should remove them from all recurring messaging campaigns unless they expressly elect to continue receiving messages from a particular program.
```

Comment:
Suggest removing "For example " and re-phrasing this to:
One of the most common opt-out mechanisms in SMS applications is to include a “STOP”...